vite.config.js

```diff
@@ -10,7 +10,7 @@ export default {
   plugins: [
     vue(),
     copy({
-      assets: ["node_modules/monaco-editor/min/vs"],
+      assets: ["node_modules/monaco-editor/min/vs", "node_modules/intro.js/minified"],
```

Comment:
I can apparently get away without that. I'll try getting rid of it, please report back if my changes work or if they horribly break everything.

Reply:
works fine for me, I wasn't quite sure at first tho, so I did that just to be sure